site/content/en/docs/tasks/run_plain_pods.md

```diff
@@ -40,12 +40,26 @@ This guide is for [batch users](/docs/tasks#batch-user) that have a basic unders
            values: [ "true", "True", "yes" ]
    ```
 
-2. Pods that belong to other API resources managed by Kueue are excluded from being queued by `pod` integration. 
+2. Kueue will run webhooks for all created pods if pod integration is enabled. Webhook namespaceSelector could be 
+   used to filter the pods to reconcile. Default webhook namespaceSelector is:
+   ```yaml
+   matchExpressions:
+   - key: kubernetes.io/metadata.name
+     operator: NotIn
+     values: [ kube-system, kueue-system ]
+   ```
+   
+   In case of [installation via Helm](/docs/installation/#install-via-helm), the webhook namespace selector 
+   will match the `integrations.podOptions.namespaceSelector` in the `values.yaml`.
+   
+   Please keep in mind that namespaceSelector should never match the kueue namespace.
+
+3. Pods that belong to other API resources managed by Kueue are excluded from being queued by `pod` integration. 
    For example, pods managed by `batch/v1.Job` won't be managed by `pod` integration.
 
-3. Kueue will inject a `kueue.x-k8s.io/managed=true` label to indicate which pods are managed by it.
+4. Kueue will inject a `kueue.x-k8s.io/managed=true` label to indicate which pods are managed by it.
```

Comment:
This is outside of the scope of "before you begin".

Let's move it to the Pod definition section?